internal/consensus/common_test.go

```diff
@@ -496,6 +496,13 @@ func randStateWithApp(nValidators int, app abci.Application) (*State, []*validat
 	return randStateWithAppImpl(nValidators, app, c)
 }
 
+func randStateWithAppPbts(nValidators int) (*State, []*validatorStub) {
```

Comment:
I think that this must be the default. See https://github.com/cometbft/cometbft/pull/2329.

Namely, if we want to run **without** PBTS we need to set enableHeight to 0 for that test.

Reply:
Good point. Done.